pkg/kv/kvserver/rditer/testdata/TestReplicaDataIterator/r1/all/exclude-user/output

```diff
@@ -0,0 +1,24 @@
+| /Local/RangeID/1/{u""-v""}             | 0169897572667462                                             |              |                                    | /Local/RangeID/1/u/RangeTombstone                                                  |
+| /Local/RangeID/1/{u""-v""}             | 0169897572667468                                             |              |                                    | /Local/RangeID/1/u/RaftHardState                                                   |
+| /Local/RangeID/1/{u""-v""}             | 016989757266746c0000000000000001                             |              |                                    | /Local/RangeID/1/u/RaftLog/logIndex:1                                              |
+| /Local/RangeID/1/{u""-v""}             | 016989757266746c0000000000000002                             |              |                                    | /Local/RangeID/1/u/RaftLog/logIndex:2                                              |
+| /Local/RangeID/1/{u""-v""}             | 01698975726c7274                                             |              |                                    | /Local/RangeID/1/u/RangeLastReplicaGCTimestamp                                     |
+| /Local/RangeID/1/{u""-v""}             | 016989753a61                                                 | 016989753a78 | 000000000000000109                 | /Local/RangeID/1/u":{a"-x"}/0.000000001,0                                          |
+| /Local/Range"{a"-b"}                   | 016b1261000172647363                                         |              | 0000000000000001                   | /Local/Range"a"/RangeDescriptor/0.000000001,0                                      |
+| /Local/Range"{a"-b"}                   | 016b1261000174786e2d0ce61c175eb445878c36dcf4062ada4c         |              |                                    | /Local/Range"a"/Transaction/"0ce61c17-5eb4-4587-8c36-dcf4062ada4c"                 |
+| /Local/Range"{a"-b"}                   | 016b126100ff000174786e2d9855a1ef8eb94c06a106cab1dda78a2b     |              |                                    | /Local/Range"a\x00"/Transaction/"9855a1ef-8eb9-4c06-a106-cab1dda78a2b"             |
+| /Local/Range"{a"-b"}                   | 016b1261ffffffff000174786e2d295e727c8ca9437cbb5e8e2ebbad996f |              |                                    | /Local/Range"a\xff\xff\xff\xff"/Transaction/"295e727c-8ca9-437c-bb5e-8e2ebbad996f" |
+| /Local/Lock/Intent/Local/Range"{a"-b"} | 017a6b12016b126100ff01726473630001                           |              | 030ce61c175eb445878c36dcf4062ada4c | /Local/Range"a"/RangeDescriptor                                                    |
+| /Local/Lock/Intent"{a"-b"}             | 017a6b12610001                                               |              | 030ce61c175eb445878c36dcf4062ada4c | "a"                                                                                |
```

Comment:
Yes, this is expected (and is the same as the previous behaviour of this test too). It's because we decode the lock table key and only pretty-print the MVCC portion of the key:

https://github.com/cockroachdb/cockroach/blob/4c397610e416cd81c839ec2f8b7cbc5ac51f8322/pkg/kv/kvserver/rditer/replica_data_iter_test.go#L186